conductr_cli/sandbox_start.py

```diff
@@ -0,0 +1,159 @@
+from conductr_cli import terminal
+
+
+#############################
+###       Constants       ###
+#############################
```

Comment:
Have we been using this commenting convention elsewhere? If not then either we apply globally, or not at all.